app/static/js/request/all.js

```diff
@@ -0,0 +1,6 @@
+/**
```

Comment:
We don't have to create separate js files to activate the popovers. 
Can put:
```javascript
$(function () {
    $("[data-toggle='popover']").popover();
});
```
in main.js and remove the redundant js files containing popovers.